docs/apache-airflow/core-concepts/executor/celery.rst

```diff
@@ -29,6 +29,8 @@ change your ``airflow.cfg`` to point the executor parameter to
 For more information about setting up a Celery broker, refer to the
 exhaustive `Celery documentation on the topic <https://docs.celeryq.dev/en/latest/getting-started/>`_.
 
+The configuration parameters of the Celery Executor can be found in :doc:`apache-airflow-providers-celery:configurations-ref`.
```

Comment:
Should these be notes instead?

Reply:
I think it's better to have them as just regular paragraphe.

Reply:
This is a basic information, not some kind of "additional" thing.